prom2json.go

```diff
@@ -137,6 +138,10 @@ func FetchMetricFamilies(
 		}
 		tlsConfig.BuildNameToCertificate()
 		transport = &http.Transport{TLSClientConfig: tlsConfig}
+	} else if skipCACheck {
```

Comment:
`InsecureSkipVerify` isn't mutually exclusive with client certificates, is it?

In principle, you may want to have client auth with a client cert/key pair, while at the same time want to skip the check of the server cert.